arviz/data/inference_data.py

```diff
@@ -589,6 +593,103 @@ def to_dataframe(
             (dfs,) = dfs.values()
         return dfs
 
+    def to_zarr(self, store=None):
+        """Convert InferenceData to a :class:`zarr.hierarchy.group`.
+
+        The zarr storage is using the same group names as the InferenceData.
+
+        Raises TypeError if no valid store is found.
+
+        Parameters
+        ----------
+        store: zarr.storage i.e MutableMapping or str, optional
+            Zarr storage class or path to desired DirectoryStore.
+
+        Returns
+        -------
+        zarr.hierarchy.group
```

Comment:
I think it should be uppercase G

```suggestion
        zarr.hierarchy.Group
```